sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/UtilsImpl.java

```diff
@@ -248,6 +249,7 @@ public static void validateDigest(MessageDigest messageDigest, String requestedD
     }
 
     public static Response<GetManifestResult> toGetManifestResponse(String tagOrDigest, Response<BinaryData> rawResponse) {
+        checkManifestSize(rawResponse.getHeaders());
```

Comment:
This didn't throw here before, are we okay with that runtime change?

Reply:
good catch!  I'll add a new changelog entry - this code is still in beta